content/integrate/redis-data-integration/ingest/troubleshooting.md

```diff
@@ -0,0 +1,78 @@
+---
+Title: Ingest troubleshooting
+aliases: null
+alwaysopen: false
+categories:
+- docs
+- integrate
+- rs
+- rdi
+description: Solve and report simple problems with RDI ingest
+group: di
+hideListLinks: false
+linkTitle: Troubleshooting
+summary: Redis Data Integration keeps Redis in sync with the primary database in near
+  real time.
+type: integration
+weight: 50
+---
+
+The following sections explain how you can get extra information from
+RDI to help you solve problems that you may encounter. Redis support may
+also ask you to provide this information if you need to contact them.
+
+## Debug information during installation {#install-debug}
+
+If the installer fails with an error then try installing again with the
+log level set to `DEBUG`:
+
+```bash
+./install.sh -l DEBUG   # Installer script
+redis-di install -l DEBUG    # Install command
+```
+
+This gives you more detail about the installation steps and can often
+help you to pinpoint the source of the error.
+
+## RDI logs
+
+By default, RDI records the following logs in the host VM file system at
+`/opt/rdi/logs` (or whichever path you specified during installation);
+
+| Filename | Phase |
+| :-- | :-- |
+| `rdi_collector-collector-initializer.log` | Initializing the Debezium collector. |
+| `rdi_collector-debezium-ssl-init.log` | Establishing the Debezium SSL connections to the source and RDI database (if you are using SSL). |
+| `rdi_collector-collector-source.log` | Debezium [change data capture (CDC)]({{< relref "/integrate/redis-data-integration/ingest/architecture" >}}) operations. |
+| `rdi_rdi-rdi-operator.log` | RDI and Kubernetes deployment. |
```

Comment:
RDI operator - the RDI control plane main component

Reply:
Fixed.